test/stdcpp/win64.mak

```diff
@@ -5,29 +5,34 @@ MODEL=64
 DRUNTIMELIB=druntime64.lib
 CC=cl
 
-_MSC_VER = $(file < ..\..\ver.txt)
+TESTS=array allocator new string
 
-TESTS= array allocator new string
+_MSC_VER=$(file < ..\..\ver.txt)
+ADD_CFLAGS=$(file < ..\..\cflags.txt)
+ADD_DFLAGS=$(file < ..\..\dflags.txt)
+ADD_TESTS=$(file < ..\..\add_tests.txt)
+
+TESTS=$(TESTS) $(ADD_TESTS)
 
 test: $(TESTS)
 
 $(TESTS):
-	"$(CC)" -c /Fo$@_cpp.obj test\stdcpp\src\$@.cpp /EHsc /MT
-	"$(DMD)" -of=$@.exe -m$(MODEL) -conf= -Isrc -defaultlib=$(DRUNTIMELIB) -main -unittest -version=_MSC_VER_$(_MSC_VER) -mscrtlib=libcmt test\stdcpp\src\$@_test.d $@_cpp.obj
+	"$(CC)" -c /Fo$@_cpp.obj test\stdcpp\src\$@.cpp /EHsc /MT $(ADD_CFLAGS)
```

Comment:
Maybe this also works directly if there is a file with the name ADD_CFLAGS that has the settings, see https://github.com/DigitalMars/Compiler/blob/1dbd3e3381c8f7f7ab2e35214dcd63455ae38c29/dm/src/make/dmake.d#L1130.

Reply:
I didn't see any documentation to that effect in the scant docs.
That would be highly unusual though I think... I kinda like that way I've written it makes it terribly clear what mischief is at play.